README.md

```diff
@@ -1 +1,5 @@
-My name is Alex, I have been working with mentally disbaled populations for the past few years. I am fairly new to programming though I hae been messing with it casually for a while now. I'm a little nervous about the workload but I amexcited to learn something new.
```

Comment:
Missing a few sections. Your `answer-two` branch should have had "Profiles" links and "Technical Skills" list as listed in the slides.